File: test/fixtures/csharp/get_basic_auth.cs
using System.Net.Http;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "http://localhost:28139/");

request.Headers.Add("Authorization", "Basic " + Convert.ToBase64String(System.Text.ASCIIEncoding.ASCII.GetBytes("some_username:some_password")));

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();

File: test/fixtures/java/get_basic_auth.java
import java.io.IOException;
import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;
import java.util.Base64;

HttpClient client = HttpClient.newHttpClient();

String credentials = "some_username" + ":" + "some_password";
String auth = "Basic " + Base64.getEncoder().encodeToString(credentials.getBytes());

HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://localhost:28139/"))
    .GET()
    .setHeader("Authorization", auth)
    .build();

HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
